tlslite/recordlayer.py

```diff
@@ -437,7 +459,8 @@ def _encryptThenSeal(self, buf, contentType):
         buf = self._writeState.encContext.seal(nonce, buf, authData)
 
         #AES-GCM, has an explicit variable nonce.
-        if "aes" in self._writeState.encContext.name and self.version < (3, 4):
+        if "aes" in self._writeState.encContext.name and \
```

Comment:
That wouldn't handle camellia-gcm or CCM ciphersuites.

Reply:
camellia and AES-CCM is completely unsupported now, so it's not an issue, we'll cross that bridge when we get to it